oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/PipelinedMongoDownloadTask.java

```diff
@@ -129,6 +133,117 @@ public long getDocumentsDownloaded() {
 
     private static final String THREAD_NAME = "mongo-dump";
 
+    /**
+     * Creates the filter to be used in the Mongo query
+     *
+     * @param mongoFilterPaths         The paths to be included/excluded in the filter. These define subtrees to be included or excluded.
+     *                                 (see {@link MongoFilterPaths} for details)
+     * @param customExcludedPathsRegex A custom regex pattern to exclude paths from the download.
```

Comment:
I would add:

    This is typically a _suffix_, for example "/metadata.xml$|/renditions/.*.jpg$".
    To exclude subtrees such as /content/abc, use mongoFilterPaths instead.